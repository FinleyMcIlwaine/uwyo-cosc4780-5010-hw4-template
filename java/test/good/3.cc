// Macros to shut up the C++ warnings
#define printInt(x)
#define printDouble(x)
#define printString(x)
#define readInt() 0
#define readDouble() 0
#define readString() 0

int main()
{
    int i = readInt();

    printInt(i);
    printInt(i++);
    printInt(i);
    printInt(++i);
    printInt(i);
}
