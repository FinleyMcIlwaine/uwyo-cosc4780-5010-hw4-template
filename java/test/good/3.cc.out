24
24
25
26
26
